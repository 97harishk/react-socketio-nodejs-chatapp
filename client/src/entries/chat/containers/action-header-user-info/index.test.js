import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	ActionHeaderUserInfo
} from 'entries/chat/containers';

describe('<ActionHeaderUserInfo />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<ActionHeaderUserInfo {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
