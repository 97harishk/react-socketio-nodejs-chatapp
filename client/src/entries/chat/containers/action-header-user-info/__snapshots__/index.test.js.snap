// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionHeaderUserInfo /> renders correctly 1`] = `
<div>
  <ProfilePictureComponent
    backgroundColor="#1461ff"
    color="white"
    height={60}
    label="AM"
    width={60}
  />
  <LabelComponent
    dark={true}
    fontSemiBold={true}
    fontSize={16}
    margin="10px 0px 0px 0px"
    text="Amendowins"
  />
</div>
`;
